etc/schema.json

```diff
@@ -331,6 +331,40 @@
             },
             "additionalProperties": false
         },
+        "arp": {
+            "type": "object",
+            "optional": true,
+            "properties": {
+                "hw_type": {
+                    "type": "string"
```

Comment:
Oh, new schema entries should come with a "description" please, for some auto generated documentation that is in the works.